src/views/Portfolio.vue

```diff
@@ -117,9 +117,27 @@
                         <img src="@/assets/images/arrow-right.svg" class="cursor-pointer" />
                     </td>
                 </tr>
+                <tr v-if="loading" class="border-b bg-main cursor-pointer basic-hover">
+                    <td class="px-4 py-2 font-semibold flex">
+                        <div class="rounded-full animate-pulse bg-white w-6 h-6" />
+                        <div class="animate-pulse bg-white pl-2 w-full">.</div>
+                    </td>
+                    <td><div class="animate-pulse bg-white">.</div></td>
+                    <!-- <td><div class="animate-pulse bg-white">.</div></td> -->
+                    <td><div class="animate-pulse bg-white">.</div></td>
+                    <td><div class="animate-pulse bg-white">.</div></td>
+                    <td>
+                        <div class="animate-pulse bg-white">
+                            <span class="text-white">.</span>
+                        </div>
+                    </td>
+                    <td class="pr-2 m-0">
+                        <div class="rounded-full animate-pulse bg-white w-6 h-6" />
+                    </td>
+                </tr>
```

Comment:
this looks nice but its too close to eachother, adding some extra space after the first item and making sure all spaces are equal will make it look better